sphinx/templates/quickstart/conf.py_t

```diff
@@ -117,6 +117,9 @@ html_static_path = ['{{ dot }}static']
 #
 # html_sidebars = {}
 
+# Cross-reference :manpage: roles to actual sites.
+#
+# manpage_url_pattern = 'https://manpages.debian.org/%s
```

Comment:
To make default `conf.py` simple, please do not add new comment-block her.

Reply:
sorry, what do you mean? should i add it elsewhere? or not at all? i added this because it says that should be done on top of the Config class.

Reply:
Oh, sorry. Please forget the comment of Config class. I will change it later.
Current `conf.py` contains a lot of comments. I think it confuses users. So we have a plan to reduce these comments from `conf.py`.

Reply:
Could you remove this please?

Reply:
done. it wasn't clear from your comment you actually wanted it removed (or if you would remove it yourself at a later point).